plugins/amazonq/mynah-ui/src/mynah-ui/ui/connector.ts

```diff
@@ -340,4 +360,25 @@ export class Connector {
                 break
         }
     }
+
+    onFormButtonClick = (
+        tabId: string,
+        messageId: string,
+        action: {
+            id: string
+            text?: string
+            formItemValues?: Record<string, string>
+        }
+    ) => {
+        if (
+            action.id === 'codetransform-input-confirm'
```

Comment:
NIT: Can we have an enum value for all the action 'ids'?

Reply:
the kotlin-side handling should also have generated types at some point

Reply:
Converted to using enum.